mypy/config_parser.py

```diff
@@ -100,7 +122,22 @@ def check_follow_imports(choice: str) -> str:
     'cache_dir': expand_path,
     'python_executable': expand_path,
     'strict': bool,
-}  # type: Final
+}  # type: Final[Dict[str, Any]]
+
+# Reuse the ini_config_types and overwrite the diff
+toml_config_types = copy.deepcopy(ini_config_types)  # type: Final[Dict[str, Any]]
```

Comment:
This seems to be the issue. Copying a compiled lambda doesn't work because of a mypyc bug: https://github.com/mypyc/mypyc/issues/804#issuecomment-802947572

Can you try replacing this with `copy.copy`? It should work.